Number.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Number {
+    //Напишіть метод, який ділить кожен елемент масива на задане число і виводить на
+    //екран новий масив елементів із поділеними елементами попереднього. Додайте
+    //опрацювання виняткових ситуацій
+        public static void main(String[] args) {
+        int[] array = {10, 15, 25};
```

Comment:
According to the home task, it's required to create a new method. We just call methods in the main method.

Please review implementation:

public static void divideArrayElements(int[] array, int divisor) {
        int[] dividedArray = new int[array.length];

        try {
            for (int i = 0; i < array.length; i++) {
                dividedArray[i] = array[i] / divisor;
            }
            System.out.println("Divided array elements:");
            for (int num : dividedArray) {
                System.out.println(num);
            }
        } catch (ArithmeticException e) {
            System.out.println("Error: Division by zero is not allowed.");
        } finally {
            System.out.println("Finally block executed.");
        }
    }

    public static void main(String[] args) {
        int[] array = {10, 5, 0, 20};
        int divisor = 0;

        divideArrayElements(array, divisor); //HERE WE CALL OUR METHOD divideArrayElements
    }